### PR TITLE
Add detailed statistics on instructor assessment and instructor question pages.

### LINKED ISSUE #658: Sorting of question statistics
If you guys have a minute, maybe you can fix the sorting of the question statistics? I noticed in my Final that the sorting is: "1, 10, 11, 12, ..., 20, 2, ... " Thanks very much for your help!

### PR DESCRIPTION
**Pull request by parasgithub**
close #658

### CODE REVIEW

File: sprocs/instance_questions_calculate_stats.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+BEGIN
+    WITH more_info AS (
+        SELECT
+            aq.assessment_id,
+            aq.question_id,
+            a.course_instance_id,
+            aq.id AS assessment_question_id
+        FROM
+            instance_questions AS iq
+            JOIN assessments AS a ON (a.id = iq.assessment_id)
```
Comment (mwest1066) [inline sprocs/instance_questions_calculate_stats.sql]: This fails because there is no column `instance_questions.assessment_id`.

Comment (parasgithub) [reply sprocs/instance_questions_calculate_stats.sql]: Not sure why I didn't catch this... the migration version of the code doesn't seem to have this bug.

Comment (parasgithub) [reply sprocs/instance_questions_calculate_stats.sql]: Fixed in the next commit.

**Comment by mwest1066** [comment]
Replaced by #891

**Pull request closed without merging**